src/test/java/org/apache/commons/lang3/function/MethodFixtures.java

```diff
@@ -124,7 +124,7 @@ public static String staticGetString() {
     @AfterEach
     public void clear() {
```

Comment:
I don't think this method is needed at all. Whoever wrote it probably had a common misunderstanding about how JUNit works. Every test method gets a completely new object. There's no need to reset fields